src/test/java/de/doubleslash/keeptime/common/ConfigParserTest.java

```diff
@@ -0,0 +1,192 @@
+      when(controller.getAvailableProjects()).thenReturn(projects);
+      when(spyModel.size()).thenReturn(4);
+      doAnswer(new Answer() {
+         @Override
+         public Object answer(final InvocationOnMock invocation) {
+            final Object[] args = invocation.getArguments();
+            final Object mock = invocation.getMock();
+            Log.debug("called with arguments: " + Arrays.toString(args));
+            projects.add(new Project((String) args[0], (Color) args[2], (Boolean) args[1], (Integer) args[3]));
+            return "";
+
+         }
+      }).when(controller).addNewProject(anyString(), anyBoolean(), any(Color.class), anyInt());
+
+      loadConfigFile(CONFIG_FILENAME);
+
+      verify(controller).addNewProject(eq("Carl Zeiss"), anyBoolean(), any(Color.class), anyInt());
+      verify(controller).addNewProject(eq("Peter Lustig"), anyBoolean(), any(Color.class), anyInt());
+
+      assertEquals(6, projects.size());
+   }
+
+   private void loadConfigFile(final String configFile) {
+      final ConfigParser testSubject = new ConfigParser(model, controller);
+      final File inputfile = new File(configFile);
+      testSubject.parserConfig(inputfile);
+   }
+
+   @Test
+   public void testDoNotImportIdle() {
+      when(controller.getAvailableProjects()).thenReturn(projects);
+      when(spyModel.size()).thenReturn(4);
+      doAnswer(new Answer() {
+         @Override
+         public Object answer(final InvocationOnMock invocation) {
+            final Object[] args = invocation.getArguments();
+            final Object mock = invocation.getMock();
+            Log.debug("called with arguments: " + Arrays.toString(args));
+            projects.add(new Project((String) args[0], (Color) args[2], (Boolean) args[1], (Integer) args[3]));
+            return "";
+
+         }
+      }).when(controller).addNewProject(anyString(), anyBoolean(), any(Color.class), anyInt());
+
+      loadConfigFile(CONFIG_FILENAME);
+
+      verify(controller, never()).addNewProject(eq("Idle"), anyBoolean(), any(Color.class), anyInt());
+   }
+
+   @Test
+   public void testImportNonexistingProjects() {
+      when(controller.getAvailableProjects()).thenReturn(projects);
+      when(spyModel.size()).thenReturn(4);
+      doAnswer(new Answer() {
+         @Override
+         public Object answer(final InvocationOnMock invocation) {
+            final Object[] args = invocation.getArguments();
+            final Object mock = invocation.getMock();
+            Log.debug("called with arguments: " + Arrays.toString(args));
+            projects.add(new Project((String) args[0], (Color) args[2], (Boolean) args[1], (Integer) args[3]));
+            return "";
+
+         }
+      }).when(controller).addNewProject(anyString(), anyBoolean(), any(Color.class), anyInt());
+
+      loadConfigFile(CONFIG_FILENAME);
+
+      verify(controller, atLeastOnce()).addNewProject(eq("Peter Lustig"), anyBoolean(), any(Color.class), anyInt());
```

Comment:
I wonder if this would return true, if the addNewProjects was not called in this test, as it was called in another test before. As you use the same controller instance for all tests? I think you should create a new mock-controller for each test case e.g. with @Before